oci/oci.go

```diff
@@ -24,6 +24,8 @@ import (
 const (
 	// ContainerStateCreated represents the created state of a container
 	ContainerStateCreated = "created"
+	// ContainerStatePaused represents the paused state of a container
+	ContainerStatePaused = "paused"
```

Comment:
I think we need special handling for this case in the stop (and maybe rm) code. You aren't allowed to stop a paused container, but the code in libkpod/stop.go only checks that the container is not already stopped, and would allow you to stop a paused container

Reply:
I've added to stop and rm both.  I'm on the fence for rm about using this error or just removing.  I decided to add and we can adjust if we get feedback saying that extra step is not wanted.